linera-views/src/graphql.rs

```diff
@@ -0,0 +1,853 @@
+// Copyright (c) Zefchain Labs, Inc.
```

Comment:
Oh yes, I did this then removed it to ease debugging 😅 I'll put it back.

Reply:
This is actually more difficult now because of the dependencies on the code in other modules.  I'll leave it for a separate PR to avoid making this one even bigger: https://github.com/linera-io/linera-protocol/issues/1327.